gulpfile.js

```diff
@@ -487,19 +493,28 @@ gulp.task('release-dist', ['build-dist'], function(done) {
         if (platformIsActive(osArch)) {
             switch (osArch) {
                 case 'win-ia32':
-                    shell.cp(path.join(distPath, 'win-ia32', `${applicationName} Setup ${version}-ia32.exe`), releasePath);
+                    shell.cp(path.join(distPath, 'win-ia32', `${applicationName} Setup ${version}-ia32.exe`),
+                            path.join(releasePath, `${applicationName.replace(/\s/, '-')}-win32-${version.replace(/\./g, '-')}.exe`));
                     break;
                 case 'win-x64':
-                    shell.cp(path.join(distPath, 'win', `${applicationName} Setup ${version}.exe`), releasePath);
+                    shell.cp(path.join(distPath, 'win', `${applicationName} Setup ${version}.exe`), path.join(releasePath, 
+                            `${applicationName.replace(/\s/, '-')}-win64-${version.replace(/\./g, '-')}.exe`));
                     break;
                 case 'mac-x64':
-                    shell.cp(path.join(distPath, 'mac', `${applicationName}-${version}.dmg`), releasePath);
+                    shell.cp(path.join(distPath, 'mac', `${applicationName}-${version}.dmg`), 
+                            path.join(releasePath, `${applicationName.replace(/\s/, '-')}-macosx-${version.replace(/\./g, '-')}.dmg`));
```

Comment:
Both `applicationName.replace(/\s/, '-')` and `applicationName.replace(/\s/, '')` should only be calculated once and then re-used where needed, to improve code clarity and reduce errors.